src/testing/testing-utils.ts

```diff
@@ -45,18 +45,23 @@ export function expectFiles(fs: d.InMemoryFileSystem, filePaths: string[]): void
   }
 }
 
-export function doNotExpectFiles(fs: d.InMemoryFileSystem, filePaths: string[]) {
-  filePaths.forEach((filePath) => {
-    try {
-      fs.sys.statSync(filePath);
-    } catch (e) {
-      return;
-    }
+/**
+ * Testing utility to validate the non-existence of some provided file paths using a specific file system
+ *
+ * @param fs the file system to use to validate the non-existence of some files
+ * @param filePaths the paths to validate
+ * @throws when one or more of the provided file paths is found
+ */
+export function doNotExpectFiles(fs: d.InMemoryFileSystem, filePaths: string[]): void {
```

Comment:
I originally was going to keep it (and it's compliment, `expectFiles`) the same, to avoid churn in the codebase.  Looking at the places it's actually used, I don't think it's that bad (the number of lines renaming them affects').  I renamed `doNotExpectFiles` in https://github.com/ionic-team/stencil/pull/3435/commits/b8a92ff8dd68e331b6c7110cb09e0dba3217eb0c and `expectFiles` in https://github.com/ionic-team/stencil/pull/3435/commits/170c1901c055c5254e8892ddabccec81666a07ec.